pkg/cloud/azure/services/virtualmachines/virtualmachines.go

```diff
@@ -413,3 +445,84 @@ func generateImagePlan(image machinev1.Image) *compute.Plan {
 		Product:   to.StringPtr(image.Offer),
 	}
 }
+
+func generateDataDisks(vmSpec *Spec) ([]compute.DataDisk, error) {
+	dataDiskLuns := make(map[int32]struct{})
```

Comment:
Suggest to rename this maps. As far as i understand it uses only for checking presence of name and lun num in previously handled objects, can we use smth like `seenLuns` and `seenNames`?

Reply:
Yes, that's a nice idea, done.